src/mon/NVMeofGwTypes.h

```diff
@@ -131,15 +131,18 @@ struct NvmeGwMonState {
   //ceph entity address allocated for the GW-client that represents this GW-id
   entity_addrvec_t addr_vect;
   uint16_t beacon_index = 0;
+  bool failover_support;
```

Comment:
*Always* initialize members inline where possible.  This commit fails to initialize this at all in the the default constructor.